docs/usage.md

```diff
@@ -149,10 +152,11 @@ You will need to create a file with information about the samples in your experi
 | Column          | Description                                                                                                                |
 |-----------------|----------------------------------------------------------------------------------------------------------------------------|
 | `sample`        | Sample name without spaces.                                                                                                |
-| `fastq`         | Full path to FastQ file if previously demultiplexed. File has to be zipped and have the extension ".fastq.gz" or ".fq.gz". |
+| `sample_path`   | Full path to FastQ file if previously demultiplexed or to BAM file if previously aligned. FastQ File has to be zipped and have the extension ".fastq.gz" or ".fq.gz". BAM file has to have the extension ".bam". |
```

Comment:
```suggestion
| `input_file`   | Full path to FastQ file if previously demultiplexed or to BAM file if previously aligned. FastQ File has to be zipped and have the extension ".fastq.gz" or ".fq.gz". BAM file has to have the extension ".bam". |
```

Reply:
Needs to be changed everywhere.

Reply:
sample_path has been changed to input_file.